plugins/uberdev/lib/review-fences.sh

```
@@ -0,0 +1,856 @@
+# review-fences.sh -- the cross-fence helpers of commands/review-pr.md.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)